### DISCUSSION TITLE
Add support for the Cosmos-based chain

### DISCUSSION THREAD

**Discussion by ithinco** [Web3Auth/Web3Auth#720] [Feature Requests]
Could you please add support for the Cosmos-based chain? Thanks

**Comment by shahbaz17** [comment]
Hey @ithinco 

Thank you for the request. We are working on it. 

Meanwhile, can you share more info for the cosmos chain if you were to add it without Web3Auth, how would you have done it? Any docs or example links would also work.

Then we'll help you add web3Auth to it.
